packages/jaeger-ui/src/components/DependencyGraph/DAG.test.js

```diff
@@ -63,8 +64,10 @@ describe('<DAG>', () => {
         parent: 'parent-id',
       },
     ];
-    render(<DAG serviceCalls={serviceCalls} />);
-    expect(screen.getByTestId('cy')).toBeDefined();
+
+    const wrapper = shallow(<DAG serviceCalls={serviceCalls} />);
+
+    expect(wrapper.exists('.DAG')).toBe(true);
```

Comment:
can we do something smarter, like count how many nodes and edges are displayed?

Reply:
Yes, that will be better